### ISSUE TITLE
Weird translatable strings

### ISSUE THREAD

**Issue by scriptum** [geany/geany-plugins#248]
### geanypg - translatable author

```
 #: ../geanypg/src/geanypg.c:39
 msgid "Hans Alves <alves.h88@gmail.com>"
```
Is it really needed for author to be translatable?

### Just numbers

```
#: ../scope/data/scope.glade.h:51
msgid "_1"
msgstr "_1"

#: ../scope/data/scope.glade.h:52
msgid "_2"
msgstr "_2"

#: ../scope/data/scope.glade.h:53
msgid "_4"
msgstr "_4"

#: ../scope/data/scope.glade.h:54
msgid "_8"
msgstr "_8"
```

Tough in some languages, e.g. I found that:

```
./po/it.po:6062:#: ../scope/data/scope.glade.h:54
./po/it.po-6063-msgid "_8"
./po/it.po-6064-msgstr "_"
```

### Empty formatters

```
#: ../scope/src/break.c:1250
#, c-format
msgid ", %s"
```

And others:

```
#: ../scope/src/debug.c:115 ../scope/src/debug.c:120 ../scope/src/utils.c:37
#, c-format
msgid "%s: %s."

#: ../scope/src/debug.c:598 ../scope/src/thread.c:93
#, c-format
msgid "%s."

#: ../scope/src/utils.c:45
#, c-format
msgid "%s: %s"

#: ../webhelper/src/gwh-settings.c:804
#, c-format
msgid "%s:"

#: ../geanygendoc/src/ggd-file-type-loader.c:871
#, c-format
msgid "%s:%u:%u: %s"
```

### Symbols

```
#: ../scope/data/scope.glade.h:117
msgid "@"

#: ../scope/data/scope.glade.h:115
msgid "-"

#: ../scope/src/inspect.c:66
msgid "..."

#: ../scope/src/views.c:526
msgid "…"
```

P.S. Searched using multiline regexp `msgid (.+)\nmsgstr \1`

**Comment by b4n** [comment]
> ### geanypg - translatable author

Odd indeed

> ### Just numbers

A bit odd too, but I could imagine someone wanting to translate some of them, e.g. for languages having other alphabets including numbers (I don't know but e.g. Chinese, Japanese or Hebrew might fall in this category).

> Tough in some languages, e.g. I found that:
>
>```
>./po/it.po:6062:#: ../scope/data/scope.glade.h:54
>./po/it.po-6063-msgid "_8"
>./po/it.po-6064-msgstr "_"
>```

This however looks plain wrong

> ### Empty formatters

While they might be suboptimal, and surely should have a specific comment explaining what they mean to the translators, they do make sense.  E.g. in French typography you want to put a non-breakable space before colons (`:`) in a sentence, so we should have a way to do that.
But yeah, they probably would deserve checking and adding comments.

> ###Symbols

Those are very weird indeed.

**Comment by frlan** [comment]
These are not the only weird translations. During my work on German translation I found a huge number of at least unclear strings containing. To be fair, I didn't report or fix most of them just grumbling into my beard. 

However, I would leave this open as an 1.26 issue "Revisitng strings of g-p project"

**Comment by b4n** [comment]
> However, I would leave this open as an 1.26 issue "Revisitng strings of g-p project" 

Agreed.  I have some other griefs against the current strings, like plugins having the word "plugin" in their description, and missing ellipsis on some menu items.  I worked some of them around in the French translation, but we should properly fix the original strings after 1.25.

**Comment by frlan** [comment]
@b4n Please create new issues/PR with reference to this issue and setting milestone to 1.26

**Comment by scriptum** [comment]
New author in translation strings:

```msgid "Hans Alves <alves.h88@gmail.com>"```
